test/Parse/ConditionalCompilation/avrTargetWithStdlib.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %swift -typecheck %s -verify -target avr-none-none-elf -disable-objc-interop -wmo -enable-experimental-feature Embedded
```

Comment:
Let's move this test into test/embedded/ and structure it more like the other tests there (there's a bunch of "REQUIRES")